### PR TITLE
Added release notes for v5.3.6

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 1]
```diff
@@ -0,0 +1,176 @@
+= 5.3.6 Release Notes
+
+== New Features
+
+=== AtomicRef support
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:5]:
```suggestion
=== AtomicRef Support
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+* `atomicreference_clear(name='default')`
+* `atomicreference_compare_and_set(expected:any, value:any, name='default')`
+* `atomicreference_contains(value:any, name='default')`
+* `atomicreference_destroy(name='default')`
+* `atomicreference_get(name='default')`
+* `atomicreference_get_and_set(value:any, name='default')`
+* `atomicreference_is_null(name='default')`
+* `atomicreference_set(value:any, name='default')`
+
+=== Viridian advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:29]:
```suggestion
=== Viridian Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+Add `viridian_custom_class` type with the following attributes:
+
+* `id`
+* `name`
+* `status`
+* `temporary_id`
+* `generated_filename`
+
+=== Job advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:71]:
```suggestion
=== Job Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+* `job_submit(path:string, params:string*, name='', snapshot='', class_name='')`
+* `job_list(include_sql=False, include_user_cancelled=False) -> list(dict)`
+* `job_cancel(job_name_or_id:string, force=False)`
+* `job_restart(job_name_or_id:string, force=False)`
+* `job_suspend(job_name_or_id:string, force=False)`
+* `job_resume(job_name_or_id:string)`
+* `job_export_snapshot(job_name_or_id:string, name:string, cancel=False)`
+
+=== Snapshot advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:81]:
```suggestion
=== Snapshot Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+* `snapshot_list() -> list(dict)`
+* `snapshot_delete(name:string)`
+
+=== Configuration advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:86]:
```suggestion
=== Configuration Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+* `config_add(config_name_or_path:string, config:dict) -> path`
+* `config_list() -> list(config_name)`
+
+=== List advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:91]:
```suggestion
=== List Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+* `list_add(value:any, name='default', index=0)`
+* `list_clear(name='default')`
+* `list_contains(value:any, name='default')`
+* `list_destroy(name='default')`
+* `list_remove_index(index:int, name='default')`
+* `list_remove_value(value:any, name='default')`
+* `list_set(index:int, value:any, name='default')`
+* `list_size(name='default')`
+
+=== MultiMap advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:102]:
```suggestion
=== MultiMap Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+* `multimap_key_set(name='default')`
+* `multimap_lock(key:any, name='default', ttl=-1)`
+* `multimap_put(key:any, value:any, name='default')`
+* `multimap_remove(key:any, name='default')`
+* `multimap_size(name='default')`
+* `multimap_try_lock(key:any, name='default', ttl=-1)`
+* `multimap_unlock(key:any, name='default')`
+* `multimap_values(name='default')`
+
+=== Queue advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:117]:
```suggestion
=== Queue Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+
+* `queue_clear(name='default')`
+* `queue_destroy(name='default')`
+* `queue_offer(value:any*, name='default')`
+* `queue_poll(name='default')`
+* `queue_size(name='default')`
+
+=== Topic advanced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:125]:
```suggestion
=== Topic Advanced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+
+* `topic_destroy(name='default')`
+* `topic_publish(value:any+, name='default')`
+
+=== Other advanvced script functions
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:130]:
```suggestion
=== Other Advanvced Script Functions
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+* `exit(code:int)`
+* `home() -> path`
+* `version() -> string`
+* `write(value:any, location:string, format='')`
+* `read(location:string, format='default')`
+* `write_data(data:data, location:string)`
+* `read_data(location:string)`
+* `decode_data(value:data)`
+
+=== Other features
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:142]:
```suggestion
=== Other Features
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+
+* Added doc strings support to advanced scripts. All lines starting with `# doc: ` are doc strings.
+* Added `script list` command.
+* Added the `sql` advanced script command.
+* Added the `file` template action which can read from a file with one of the extensions: `.yaml`, `.json` or `.txt`.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:147]:
```suggestion
* Added the `file` template action, which can read from a file with an extension of `.yaml`, `.json` or `.txt`.
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+* Added the `\exit` and `\echo` commands. These commands are available in the script and interactive modes. See: [echo](https://docs.hazelcast.com/clc/latest/echo) and [exit](https://docs.hazelcast.com/clc/latest/exit) command documentation.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:148]:
```suggestion
* Added the `\exit` and `\echo` commands. These commands are available in the script and interactive modes. See the [echo](https://docs.hazelcast.com/clc/latest/echo) and [exit](https://docs.hazelcast.com/clc/latest/exit) command documentation.
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 14]
```diff
@@ -0,0 +1,176 @@
...
+* Added `--wait` flag to `viridian` commands.
+* Added the ability to change the interactive mode prompt and banner. See: [def](https://docs.hazelcast.com/clc/latest/def).
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:150]:
```suggestion
* Added the ability to change the interactive mode prompt and banner. See the [def](https://docs.hazelcast.com/clc/latest/def) topic.
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 15]
```diff
@@ -0,0 +1,176 @@
...
+
+== Improvements
+
+* `clc demo map-setmany` command shows a progress bar and it is faster for small values.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:154]:
```suggestion
* `clc demo map-setmany` command shows a progress bar and is faster for small values.
```

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc [hunk 16]
```diff
@@ -0,0 +1,176 @@
...
+* `\echo` command takes 0 or more arguments.
+* Viridian commands display an API key selector if the there exists more than one API keys.
```
Comment (rebekah-lawrence) [inline docs/modules/ROOT/pages/release-notes-5.3.6.adoc:156]:
```suggestion
* Viridian commands display an API key selector if more than one API key exists.
```

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]
One other thought, is it worth separating Platform and Viridian information into sections, just for ease of reference for the user?

File: docs/modules/ROOT/pages/clc-serializer.adoc [hunk 17]
```diff
@@ -56,15 +56,9 @@ clc serializer generate my-schema.yaml --language java --output-dir my-dir
...
-
-* Describe the contents of a compact class using supported field types.
-* Import other schema.
-* Specify a namespaces for schema files and reference other namespaces.
-* Reference classes that are not present in the given schemas.
-
-Schemas are written in YAML.
-
+A schema allows you to describe the contents of a compact class using supported field types.
+A schema allows you to describe the contents of a compact class using supported field types.
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/ROOT/pages/clc-serializer.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/clc-serializer.adoc [hunk 18]
```diff
@@ -56,15 +56,9 @@ clc serializer generate my-schema.yaml --language java --output-dir my-dir
...
+Schemas are written in YAML, and the schema files should have the `.schema.yaml` extension.
```
[the remark this answers is not in the document]
Comment (yuce) [reply docs/modules/ROOT/pages/clc-serializer.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 1 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 2 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 3 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 5 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 6 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 7 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 8 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 9 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 10 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 11 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 12 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 13 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 14 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 15 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

File: docs/modules/ROOT/pages/release-notes-5.3.6.adoc (hunk 16 above)
Comment (yuce) [reply docs/modules/ROOT/pages/release-notes-5.3.6.adoc]: Fixed at: https://github.com/hazelcast/clc-docs/pull/13/commits/9febcbe07c44f071e16d557213df687261458d89

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Comment by yuce** [comment]
@rebekah-lawrence  Thanks a lot for the quick review!

**Pull request merged**